decisiontree/templates/tree/answer.html

```diff
@@ -1,11 +1,13 @@
 {% extends "tree/base.html" %}
 
-{% block content %}
 {% load i18n %}
+{% load tree_tags %}
+
+{% block content %}
 <form action="" method="POST" class="pretty"> {% csrf_token %}
 <fieldset>
 <legend>
-{% if answerid %}
+{% if object %}
```

Comment:
Not related to this PR, but some of these templates could benefit from some indentation.